regression-tests/Makefile.compile-test

```diff
@@ -36,12 +36,13 @@ nine := x x x x x x x x x
 max = $(subst xx,x,$(join ${1},${2}))
 gt = $(filter-out $(words ${1}),$(words $(call max,${1},${2})))
 addzero = $(if $(call gt,${nine},$(1)),$(words ${1}),0$(words ${1}))
+get_target = $(firstword $(subst :, ,$1))
+get_target_vars = $(wordlist 2,15,$(subst :, ,$1))
 
 define dooneexample
 @echo Building example $(3): $(1) for target $(2)
 @((cd $(EXAMPLESDIR)/$(1); \
- export STM32W_CPUREV=CC; \
- make TARGET=$(2) clean && make TARGET=$(2)) > \
+ make $(4) TARGET=$(2) clean && make $(4) TARGET=$(2)) > \
```

Comment:
I tested this and it is not possible to call clean for target ```mbxxx``` without specifying ```STM32W_CPUREV``` (Travis failed). I think there will be more platforms that do not want to have default values to avoid building for the wrong hardware / board. I would suggest leaving it as it is now. We still need to assume that the platform will clean anything that is remaining from last compilation test for any other target. What do you think?